src/load-service.cc

```diff
@@ -539,8 +539,8 @@ service_record * dirload_service_set::load_reload_service(const char *name, serv
 
         if (!settings.env_file.empty()) {
             try {
-                read_env_file(settings.env_file.data(), false, srv_env);
-            } catch (const std::system_error &se) {
+                read_env_file(settings.env_file.data(), false, srv_env, true);
+            } catch (const std::system_error &se) { // system_error, bad_alloc
```

Comment:
This comment here doesn't make sense, this `catch` clause is only catching `system_error` so why add a comment that also says `bad_alloc`?